go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
 	github.com/opencontainers/runtime-tools v0.9.0
 	github.com/opencontainers/selinux v1.10.0
-	github.com/openshift/imagebuilder v1.2.2
+	github.com/openshift/imagebuilder v1.2.3-0.20220329111835-9afddc0c1588
```

Comment:
Ideally this would not be a pinned commit, but a tag.

Reply:
Afaik this could be only `v1.2.3`, `v1.2.3` just got released https://github.com/openshift/imagebuilder/releases/tag/v1.2.3